extensions/widgets/clock/clock.lcb

```diff
@@ -99,10 +295,72 @@ public handler OnSave(out rProperties as Array)
 	put the empty array into rProperties
 
 	put mTimeZone into rProperties["timeZone"]
+	put colorToString(mDayNumberColor,true) into rProperties["dayNumberColor"]
+	put colorToString(mNightNumberColor,true) into rProperties["nightNumberColor"]
+	put colorToString(mDayHourHandColor,true) into rProperties["dayHourHandColor"]
+	put colorToString(mNightHourHandColor,true) into rProperties["nightHourHandColor"]
+	put colorToString(mDayMinuteHandColor,true) into rProperties["dayMinuteHandColor"]
+	put colorToString(mNightMinuteHandColor,true) into rProperties["nightMinuteHandColor"]
+	put colorToString(mDaySecondHandColor,true) into rProperties["daySecondHandColor"]
+	put colorToString(mNightSecondHandColor,true) into rProperties["nightSecondHandColor"]
+	put colorToString(mDayFaceColor,true) into rProperties["dayFaceColor"]
+	put colorToString(mNightFaceColor,true) into rProperties["nightFaceColor"]
 end handler
 
 public handler OnLoad(in pProperties as Array)
 	put pProperties["timeZone"] into mTimeZone
+
+	--these properties were added in version 1.1.3
+	if "dayNumberColor" is among the keys of pProperties then
+		put stringToColor(pProperties["dayNumberColor"]) into mDayNumberColor
+	else
+		put color kBlack into mDayNumberColor
+	end if
```

Comment:
You don't need these `else` clauses - `OnCreate` is always called before `OnLoad` so the defaults will already be set there

Reply:
Ah, neat. I thought it only called OnCreate when the widget was first created - it makes more sense that it's when it's created when the stack opens and then all the properties are loaded in.

Will fix.